src/cmd/services/m3query/config/config.go

```diff
@@ -182,9 +182,6 @@ type Configuration struct {
 	// ResultOptions are the results options for query.
 	ResultOptions ResultOptions `yaml:"resultOptions"`
 
-	// Experimental is the configuration for the experimental API group.
-	Experimental ExperimentalAPIConfiguration `yaml:"experimental"`
```

Comment:
This would fail loading config in case someone has specified it. Might be better to change to something like this instead:
```
Unused1 ExperimentalAPIConfiguration `yaml:"experimental"`
```

Reply:
https://github.com/m3db/m3/pull/3726/commits/d9e6581322a4b150e7066160b43f2be987a87dc8